src/backend/tests/projects.test.ts

```diff
@@ -68,6 +68,17 @@ describe('Projects', () => {
     expect(res.statusCode).toBe(400);
   });
 
+  test('newProject fails when unknown team Id provided', async () => {
+    mockGetChangeRequestReviewState.mockResolvedValue(true);
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue({ ...batman, googleAuthId: 'b' });
```

Comment:
don't need the googleAuthId anymore i think. if you see this in other places can you take it out cuz i think one person did it now everyone thinks we need it lol

Reply:
yep